test-case/check-pause-release.sh

```diff
@@ -3,16 +3,16 @@
 set -e
 
 ##
-## Case Name: check-pause-resume
+## Case Name: check-pause-release
 ## Preconditions:
 ##    N/A
 ## Description:
-##    playback/capture on each pipeline and feak pause/resume with expect
+##    playback/capture on each pipeline and feak pause/release with expect
```

Comment:
what does feak mean?

Reply:
fake pause_push/pause_release with expect?

Reply:
The description section need to be re-written, I changed only 'fake' with 'trigger'